app/views/facilities/list.html.haml

```diff
@@ -11,6 +11,10 @@
         %th= Facility.human_attribute_name(:name).titlecase
         %th= Facility.human_attribute_name(:status).titlecase
     %tbody
+      -if current_user.administrator?
```

Comment:
Should we feature flag this for now? Hopefully, eventually we can open it up at some point, but I'm not sure I want to give this to NU/UIC just yet without making sure it doesn't kill the servers.

Reply:
What are we worrying about that would kill the servers? Happy to feature flag it, but not sure I understand exactly what the concern is.

Reply:
The products/accounts/etc tabs could eat up a huge chunk of memory and or time. I'd like to see what sort of timings we get from this once we're actually on a production machine. As long as we can still access the page via url hacking.